meta-mender-core/classes/mender-part-images.bbclass

```diff
@@ -261,6 +261,28 @@ IMAGE_TYPEDEP_biosimg_append = "${@bb.utils.contains('IMAGE_FSTYPES', 'sdimg', '
 IMAGE_TYPEDEP_gptimg_append = "${@bb.utils.contains('IMAGE_FSTYPES', 'sdimg', ' sdimg', '', d)} \
                                ${@bb.utils.contains('IMAGE_FSTYPES', 'uefiimg', ' uefiimg', '', d)} \
                                ${@bb.utils.contains('IMAGE_FSTYPES', 'biosimg', ' biosimg', '', d)}"
+# Make sure the Mender part image is available in the live installer
+IMAGE_TYPEDEP_hddimg_append = "${@bb.utils.contains('IMAGE_FSTYPES', 'sdimg', ' sdimg', '', d)} \
+                               ${@bb.utils.contains('IMAGE_FSTYPES', 'uefiimg', ' uefiimg', '', d)} \
+                               ${@bb.utils.contains('IMAGE_FSTYPES', 'biosimg', ' biosimg', '', d)}"
+
+# Use the Mender part image as the Live image
+python() {
+  if bb.utils.contains('IMAGE_FSTYPES', 'sdimg', True, False, d):
+     type='sdimg'
+  elif bb.utils.contains('IMAGE_FSTYPES', 'uefiimg', True, False, d):
+     type='uefiimg'
+  elif bb.utils.contains('IMAGE_FSTYPES', 'biosimg', True, False, d):
+     type='biosimg'
+
+  d.setVar('LIVE_ROOTFS_TYPE', type)
+  d.setVar('ROOTFS', "${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}.%s.bz2" % type)
+  d.setVar('IMAGE_FSTYPES_append', ' %s.bz2 ' % type)
+
+  # Remove the boot option on the Live installer; it won't work since Mender hard codes
+  # the device nodes
+  d.setVar('LABELS_LIVE_remove', 'boot')
+}
```

Comment:
Inconsistent indentation, both using 2 and 3 spaces. Should stick to 4, like in other Python sections.

Reply:
Fixed.